frontend/src/components/auth/Login.tsx

```diff
@@ -89,77 +132,159 @@ const Login = () => {
   }
 
   return (
```

Comment:
nit: this Login component is getting very big and becoming hard to read. Thoughts on splitting it up abit? For example
- we could move all the `FormControl` to `LoginForm`
- we could move this entire return to `LoginComponent`

What do you think? You know this code better than I do and have a sense of how the jsx could be broken up. This return is currently about 150 lines of jsx code (wow!)